src/utils/helpers/DrawHelpers.ts

```diff
@@ -86,14 +82,14 @@ export function createSVGContainer(
         .attr('height', height);
 }
 
-export function createZoom(svgContainer: NodeSelection<SVGSVGElement>): Selection<SVGGElement, DependencyNode, Element, HTMLElement> {
-    const zoomLayer = svgContainer.append('g').attr('id', 'zoom');
+export function createZoom(svgContainer: Selection<any, any, any, any>, selector: Selectors.ZOOM_OVERVIEW | Selectors.ZOOM_DETAILS) {
+    const zoomLayer = svgContainer.append('g').attr('id', selector.slice(1));
```

Comment:
Maybe it would be better to have selector constants without hash

`DETAILS_VIEW_CONTAINER = 'details-view-container'`

`export function selectDetailsZoom() {
    return select('#${Selectors.ZOOM_DETAILS}');
}`

`.attr('id', selector);
`

Reply:
yeah, that seems like a good idea